charts/grafana/FLUX.md

```diff
@@ -26,4 +26,5 @@
 - Use the [`limit()`](https://docs.influxdata.com/flux/v0.12/functions/transformations/limit/) function to reduce the number of rows queried from InfluxDB, this should be done because the current iteration of the Grafana's Flux Plugin does not handle large quantities of rows
 - [`join()`](https://docs.influxdata.com/flux/v0.12/functions/transformations/join) is not a pipe function and does not use the pipe forward operator, `join()` instead is a start point for that line of the query
 - Avoid pressing the tab key while in the editor as it will autocomplete code that may not be intended for execution
-- Leave the `from(bucket: "undefined")` pre-made query until the query is written. This makes the query run on an undefined database in InfluxDB, so development can continue without overloading the Grafana IDE
\ No newline at end of file
+- Leave the `from(bucket: "undefined")` pre-made query until the query is written. This makes the query run on an undefined database in InfluxDB, so development can continue without overloading the Grafana IDE
+- If using windowed data, the `window(every: inf)` query must be used to turn data back into a time series for use with grafana
```

Comment:
This is confusing.
Maybe say something like "Leave the [pre-made] query as ... "

Reply:
responded in my latest commit 9f204b0

Reply:
what do you mean you responded in a commit

Reply:
You also still left it confusing.

Reply:
I believe the commit 0948131 might have fixed the confusingness?